src/Exceptionless.Web/Bootstrapper.cs

```diff
@@ -1,25 +1,55 @@
 ﻿using System;
 using System.Threading.Tasks;
+using App.Metrics;
+using App.Metrics.AspNetCore;
+using App.Metrics.Formatters.Prometheus;
+using App.Metrics.Formatters;
 using AutoMapper;
 using Exceptionless.Core;
 using Exceptionless.Core.Extensions;
 using Exceptionless.Core.Jobs.WorkItemHandlers;
 using Exceptionless.Core.Models;
 using Exceptionless.Core.Models.Data;
 using Exceptionless.Core.Queues.Models;
+using Exceptionless.Insulation.Metrics;
 using Exceptionless.Web.Hubs;
 using Exceptionless.Web.Models;
 using Exceptionless.Web.Utility;
 using Exceptionless.Web.Utility.Handlers;
 using Foundatio.Jobs;
 using Foundatio.Messaging;
+using Microsoft.AspNetCore.Hosting;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.Logging;
 using Stripe;
 
 namespace Exceptionless.Web {
     public class Bootstrapper {
+        public static void ConfigureWebHost(IWebHostBuilder builder) {
+            if (!String.IsNullOrEmpty(Settings.Current.ApplicationInsightsKey))
```

Comment:
This is happening twice and should be removed. Can we just move this to an extension method?